CheatSheet/scripts/setup-supabase.sh

```
@@ -0,0 +1,156 @@
+    echo "Please ensure NEXT_PUBLIC_SUPABASE_URL and SUPABASE_SERVICE_ROLE_KEY are set in .env"
+    exit 1
+fi
+
+# Extract project ID from URL
+PROJECT_ID=$(echo $NEXT_PUBLIC_SUPABASE_URL | sed 's/.*\/\/\([^.]*\).*/\1/')
+
+echo ""
+echo -e "${BLUE}🔗 Linking to Supabase project: ${PROJECT_ID}${NC}"
+
+# Link to the project
+cd ../supabase
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)